container/common/container_hints.go

```diff
@@ -25,7 +25,7 @@ import (
 	"os"
 )
 
-var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
+var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
```

Comment:
can we just merge this and figure out what to do with this once I drop the
rkt handler PR?

On Thu, Mar 3, 2016 at 12:28 PM, Tim St. Clair <notifications@github.com>
wrote:

> In container/common/container_hints.go
> <https://github.com/google/cadvisor/pull/1128#discussion_r54943076>:
>
> > @@ -25,7 +25,7 @@ import (
> >  	"os"
> >  )
> >
> > -var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
> > +var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
>
> The test could set the flag value instead. This is pretty nit-picky though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1128/files#r54943076>.
>